src/v2/view-builder/components/HeaderBeacon.js

```diff
@@ -14,19 +14,28 @@ import hbs from 'handlebars-inline-precompile';
 
 const BeaconView = View.extend({
 
-  template: hbs('\
-    <div data-type="beacon-container" class="beacon-container">\
-      <div class="beacon-blank auth-beacon">\
-        <div class="beacon-blank js-blank-beacon-border auth-beacon-border"></div>\
-      </div>\
-      <div class="bg-helper auth-beacon auth-beacon-factor {{className}}" data-se="factor-beacon">\
-        <div class="okta-sign-in-beacon-border auth-beacon-border"></div>\
-      </div>\
-    </div >\
-  '),
+  template: hbs`
+    <div data-type="beacon-container" class="beacon-container">
+      <div class="beacon-blank auth-beacon">
+        <div class="beacon-blank js-blank-beacon-border auth-beacon-border"></div>
+      </div>
+      {{#if logoUri}}
+      <div class="bg-helper auth-beacon auth-beacon-factor custom-logo" data-se="factor-beacon" 
+        style="background-image: url('{{logoUri}}')">
```

Comment:
I see you're using a `background-image` here instead of an `<img>` tag like in the other files. Any specific reason we can't do the same in all files, either use `background-image` or the image tag? I am guessing there is something special about the Beacon that we need this background-image here, so curious what it is

Reply:
Beacon logo is bigger than previous ones and need to support any kind of SVGs from small to large ones. Hence used `background-image` as it gives more control on positioning the image also i can reuse the existing css classes. 
used `<img>` in other places because it is semantically correct, screen readers compatible and no much customization or positioning is needed as it is very small size.

As per today's discussion, we concluded that No padding/spacing should be added from SIW. I will go ahead and replace the other occurrences of `<img>` to `background-image` so that i can reuse existing classes, and have more control on customization if needed in future. Will add `role` and `aria-label` to support Screen readers.